src/app/(home)/page.tsx

```diff
@@ -1,3 +1,5 @@
+"use client";
+
```

Comment:
this line is unnecessary

Reply:
here is the problem: "You're importing a component that needs useState. It only works in a Client Component but none of its parents are marked with "use client", so they're Server Components by default."

Reply:
all files you import in this file don't contain the line "use client" and don't use useState hook

Reply:
maybe you meant this file?
src/app/(home)/layout.tsx

Reply:
*facepalm* I'm sorry